packages/protocol/protocol_test.go

```diff
@@ -639,27 +677,617 @@ func TestEngine_ShutdownResume(t *testing.T) {
 		"Z": identity.GenerateIdentity().PublicKey(),
 	}
 
-	identitiesWeights := map[identity.ID]uint64{
-		identity.New(identitiesMap["A"]).ID(): 25,
-		identity.New(identitiesMap["B"]).ID(): 25,
-		identity.New(identitiesMap["C"]).ID(): 25,
-		identity.New(identitiesMap["D"]).ID(): 25,
-		identity.New(identitiesMap["Z"]).ID(): 0,
+	identitiesWeights := map[ed25519.PublicKey]uint64{
+		identity.New(identitiesMap["A"]).PublicKey(): 25,
+		identity.New(identitiesMap["B"]).PublicKey(): 25,
+		identity.New(identitiesMap["C"]).PublicKey(): 25,
+		identity.New(identitiesMap["D"]).PublicKey(): 25,
+		identity.New(identitiesMap["Z"]).PublicKey(): 0,
```

Comment:
Not necessarily part of this PR but this code is repeated over and over again. It should be part of some testframework to make it easy to initialize a bunch of identities with alias and weights from which the snapshot is created